notebooks/Ensemble parameter exploration.ipynb

```diff
@@ -0,0 +1,1124 @@
+       "7                         0                         1                 0   \n",
+       "8                         1                         0                 0   \n",
+       "9                         1                         0                 0   \n",
+       "\n",
+       "   bert_SVM_bert_210221  bert_naive_bayes_scibert_210221  \\\n",
+       "0                     1                                1   \n",
+       "1                     0                                1   \n",
+       "2                     0                                1   \n",
+       "3                     0                                1   \n",
+       "4                     0                                1   \n",
+       "5                     1                                1   \n",
+       "6                     0                                1   \n",
+       "7                     0                                1   \n",
+       "8                     0                                1   \n",
+       "9                     0                                1   \n",
+       "\n",
+       "   count_log_reg_210221  count_naive_bayes_210221  tfidf_log_reg_210221  \\\n",
+       "0                     1                         1                     1   \n",
+       "1                     0                         1                     0   \n",
+       "2                     0                         0                     0   \n",
+       "3                     0                         0                     0   \n",
+       "4                     1                         1                     1   \n",
+       "5                     1                         1                     1   \n",
+       "6                     1                         1                     0   \n",
+       "7                     0                         0                     0   \n",
+       "8                     1                         0                     0   \n",
+       "9                     1                         0                     0   \n",
+       "\n",
+       "   bert_log_reg_scibert_210221    Internal ID  \n",
+       "0                            1  219603/Z/19/Z  \n",
+       "1                            0  219425/Z/19/Z  \n",
+       "2                            0  219420/Z/19/Z  \n",
+       "3                            0  219007/Z/19/Z  \n",
+       "4                            0  218880/Z/19/Z  \n",
+       "5                            1  218584/Z/19/Z  \n",
+       "6                            0  218539/Z/19/Z  \n",
+       "7                            0  218510/Z/19/Z  \n",
+       "8                            0  218369/Z/19/Z  \n",
+       "9                            0  218358/Z/19/Z  "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# The test predictions for each model\n",
```

Comment:
I had the impression that  you should en up with `2**12` models (each model included/not included), but the data frame only has 245. Any thoughts?

Reply:
Oh, I see, you test all the combinations below. My bad, ignore it!